src/video-interface.js

```diff
@@ -142,6 +142,18 @@ export const VideoAttributes = {
    *
    */
   AUTOPLAY: 'autoplay',
+  /**
+   * auto-fullscreen
+   *
+   * NOTE: experimental
+   * If enabled, this automatically expands the currently visible video to
```

Comment:
currently visible and `playing`.

Let's discuss with @spacedino on whether we need to exclude autoplaying from this behaviour or not (I don't see a reason to exclude autoplay from this feature)